apps/web/app/[locale]/timesheet/[memberId]/components/TimeSheetFilterPopover.tsx

```
@@ -145,9 +147,9 @@ export const TimeSheetFilterPopover = React.memo(function TimeSheetFilterPopover
 								<MultiSelect
 									localStorageKey="timesheet-select-filter-status"
 									removeItems={shouldRemoveItems}
-									items={statusOptions}
-									itemToString={(status) => (status ? status.value : '')}
-									itemId={(item) => item.value}
+									items={statusTable?.flatMap((status) => status)}
```

Biome (1.9.4)

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)